packs/feats/ceremony-of-sunlight.json

```
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p>You have learned how to master the power contained within sunlight, just like the trees that form your body. Your @UUID[Compendium.pf2e.ancestryfeatures.Item.Sunlight Healing] recovers 1d8 Hit Points per level.</p>\n<p>[[/r (@actor.level)d8]]{Sunlight Healing}</p>"
+            "value": "<p>You have learned how to master the power contained within sunlight, just like the trees that form your body. Your @UUID[Compendium.pf2e.ancestryfeatures.Item.Sunlight Healing] recovers 1d8 Hit Points per level.</p>\n<p>@Damage[(@actor.level)d8[healing]]{Sunlight Healing}</p>"
```

```suggestion
            "value": "<p>You have learned how to master the power contained within sunlight, just like the trees that form your body. Your @UUID[Compendium.pf2e.ancestryfeatures.Item.Sunlight Healing] recovers 1d8 Hit Points per level.</p>"
```
We can DamageAlteration this one to actually work automatically